### PR TITLE
Handling invalid commands

### PR DESCRIPTION
**Pull request by 0xPrateek**
#### Checklist :

  - [x] I have added the relevant documentation.
  - [x] My branch is up-to-date with the upstream master branch.
  - [x] I have fully tested the changes locally.

#### Development environment

    OS: x
    Python Version: x

### CODE REVIEW

**Comment by 0xPrateek** [comment]
OS: Parrot
Python :3

File: src/vault.py [hunk 1]
```diff
@@ -362,7 +362,11 @@ def fuzz(args):
 # Google dork
 
 def dork(args):
-    if args.dork:
+    if not args.dork:
```
Comment (abhisharma404) [inline src/vault.py]: It should be `if not args.url`, also put the import in try-catch block, take example from any other similar function.

Comment (0xPrateek) [reply src/vault.py]: Okay

File: src/vault.py [hunk 2]
```diff
@@ -362,24 +362,34 @@ def fuzz(args):
...
-                file = args.output
-            else:
-                file = args.output + '.txt'
-
-            with open(file, 'w') as f:
-                f.write('Google Dorks results: \n\n')
-                for k in web_lists:
-                    f.write(str(k) + os.linesep)
-            colors.success('File has been saved successfully')
+    if not args.dork:
```
[the remark this answers is not in the document]
Comment (0xPrateek) [reply src/vault.py]: actually google dork usage is like this
--> python3 ./vault.py -d "inurl:admin.php"
   [ SYNTAX :: vault.py -d DORK]
--> It does not require URL as an argument
hope it get clear...!

**Comment by 0xPrateek** [comment]
actually google dork usage is like this
--> python3 ./vault.py -d "inurl:admin.php"
[ SYNTAX :: vault.py -d DORK]
--> It does not require URL as an argument
hope it get clear...!

**Comment by mzfr** [comment]
@abhisharma404 	Let's close this due to inactivity.

**Pull request closed without merging**